cdm-test/src/test/java/ucar/nc2/ncml/TestNcMLStrides.java

```diff
@@ -69,8 +69,8 @@ private void testStride(String stride) throws IOException, InvalidRangeException
     Index ci = correct.getIndex();
     Index di = data.getIndex();
     for (int i = 0; i < data.getSize(); i++)
-      assert (data.get(di.set(i)) == correct.get(ci.set(i))) : stride + " index " + i + " = " + data.get(di.set(i))
-          + " != " + correct.get(ci.set(i));
+      assert (data.get(di.set(i)) == correct.get(ci.set(i)))
```

Comment:
better to change to assertThat() whenever you have to touch this. But I left NcML tests out of that conversion, so I guess you can too.

Reply:
Fixed :-)